cpp/include/raft/comms/mpi_comms.hpp

```diff
@@ -59,39 +59,39 @@
 namespace raft {
 namespace comms {
 
-constexpr MPI_Datatype get_mpi_datatype(const datatype_t datatype) {
+constexpr MPI_Datatype get_mpi_datatype(const DataTypeT datatype) {
   switch (datatype) {
-    case datatype_t::CHAR:
-      return MPI_CHAR;
-    case datatype_t::UINT8:
-      return MPI_UNSIGNED_CHAR;
-    case datatype_t::INT32:
+    case DataTypeT::kChar:
+      return MPI_kChar;
```

Comment:
Is this an incorrect sub? This constant comes from MPI, right?

Reply:
ouchy! the result of mindless regex-replace. Thanks for catching it John! Will fix it.